app/src/main/java/com/bc/gordiansigner/ui/share_account/ShareAccountMapViewModel.kt

```diff
@@ -43,7 +46,12 @@ class ShareAccountMapViewModel(
                                     if (index != -1) {
                                         keysInfo[index]
                                     } else {
-                                        KeyInfo(fingerprint, "unknown", false)
+                                        KeyInfo(
+                                            fingerprint,
+                                            "unknown",
```

Comment:
It should come with something like `KeyInfo.newUnknownInstance()`. Is it better?